lib/ibc-solidity/src/lib.rs

```diff
@@ -0,0 +1,629 @@
+pub mod ibc {
+    alloy::sol! {
+        // #![sol(cfg_attr(feature = "rpc", rpc))]
```

Comment:
do we need this?

Reply:
i have to figure out a way to properly feature gate the rpc generation, its not currently possible which is super annoying